index.js

```diff
@@ -57,14 +57,18 @@ export const SMSCalculator = {
       totalLength,
     };
   },
-  getCount: function(text) {
+  getCount: function(text, enc = '') {
     let length = text.length;
-    const encoding = this.detectEncoding(text);
+    const encoding = this.encodingLookup(enc, text)
```

Comment:
Looking at the signature, seems like it should be `this.encodingLookup(text, enc);` :)